draft-duke-quic-version-aliasing.md

```diff
@@ -104,20 +103,22 @@ entirely to the specification of the standard version.
 
 When they instantiate a connection, servers select an alternate 32-bit version
 number, and optionally an initial token extension, for the next connection at
-random and securely derive a salt and Packet Length Offset from those values
-using a repeatable process. They communicate this using a transport parameter
-extension including the version, initial token extension, salt, Packet Length
-Offset, and an expiration time for that value.
+random and securely derive a salt, packet Length Offset, and long header packet
+type codepoints from those values using a repeatable process. They communicate
+this using a transport parameter extension including the version, initial token
+extension, salt, Packet Length Offset, packet type codepoints, and an expiration
+time for that value.
```

Comment:
This list of items is getting long enough that maybe it's time to consider a single value from which the others might be derived (using HKDF, say).